container/features/src/main/resources/features.xml

```diff
@@ -1640,14 +1640,22 @@
     </feature>
 
     <feature name="opennms-endpoints-grafana" version="${project.version}" description="OpenNMS :: Features :: Endpoints :: Grafana">
+
+        <bundle dependency="true">mvn:org.jetbrains.kotlin/kotlin-osgi-bundle/${kotlinVersion}</bundle>
+        <bundle>wrap:mvn:com.google.code.gson/gson/${gsonVersion}</bundle>
+        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.okhttp/2.7.5_1</bundle>
+        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.okio/1.13.0_1</bundle>
+        <bundle dependency='true'>wrap:mvn:com.squareup.okhttp3/okhttp/3.5.0</bundle>
```

Comment:
There are multiple versions of okhttp in the same feature file.  One as wrap bundle
```
<bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.okhttp/2.7.5_1</bundle>
```
one from servicemix
```
<bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.okhttp/2.7.5_1</bundle>
```

And the original one on pom file is different. ` <okhttpVersion>4.9.3</okhttpVersion>` 

There is no clarity on what is getting used at runtime.